.github/workflows/build.yml

```diff
@@ -28,6 +28,10 @@ jobs:
       run: |
         GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' BUILD_CONTAINER_IMAGE=false build
 
+    - name: validate build
+      working-directory: /Users/runner/go/src/github.com/rook/rook
+      run: tests/scripts/validate_modified_files.sh build
+
```

Comment:
The failures caused by the racy piping were usually caught in `validate codegen`, so make sure there is a `validate build` that can catch similar issues in the future.